modin/pandas/groupby.py

```diff
@@ -116,24 +116,33 @@ def __getattr__(self, key):
     def ngroups(self):
         return len(self)
 
-    def skew(self, **kwargs):
-        return self._apply_agg_function(lambda df: df.skew(**kwargs))
+    def skew(self, *args, **kwargs):
+        return self._wrap_aggregation(
+            type(self._query_compiler).groupby_skew,
```

Comment:
I'm not happy about `type(self._query_compiler).foo`, why do we do this?.. I mean, why `self._query_compiler.foo` is not enough?

Reply:
it's a legacy of how `_wrap_aggregation` was implemented before, it calls the passed method on [modified query compiler](https://github.com/modin-project/modin/blob/34732cbd70381e99bb7c6cbcc90ac60d6f4017aa/modin/pandas/groupby.py#L952-L955), so it shouldn't be bound to the concrete qc.